Apps/Inc/SendTritium.h

```diff
@@ -1,52 +1,102 @@
 /**
- * @copyright Copyright (c) 2018-2023 UT Longhorn Racing Solar
  * @file SendTritium.h
- * @brief 
+ * @brief The SendTritium task currently houses our velocity control code. 
+ * It utilizes a FSM to coordinate what messages should be sent to the motor 
+ * controller depending on a certain set of input variables. This FSM depends 
+ * on data from across the system, such as the [Pedals](../Drivers/Pedals.html), 
+ * the [Switches](../Drivers/Minions.html), and the CAN messages from 
+ * BPS (indicating whether charging is enabled).
  * 
- * @defgroup SendTritium
- * @addtogroup SendTritium
- * @{
  */
 #ifndef __SENDTRITIUM_H
 #define __SENDTRITIUM_H
 
 #include "common.h"
 
-//#define SENDTRITIUM_PRINT_MES
+/**
+ * @def SENDTRITIUM_PRINT_MES
+ * @brief Print messages for SendTritium.c
+ */
+// #define SENDTRITIUM_PRINT_MES
+
+/**
+ * @def SENDTRITIUM_EXPOSE_VARS
+ * @brief Expose variables for SendTritium.c
+ */
+// #define SENDTRITIUM_EXPOSE_VARS
+
+/**
+ * @def MOTOR_MSG_PERIOD
+ * @brief Period of the motor CAN message in milliseconds
+ */
+#define MOTOR_MSG_PERIOD 100
+
+/**
+ * @def FSM_PERIOD
+ * @brief Period of the state machine in milliseconds
+ */
+#define FSM_PERIOD 100
 
-#define MOTOR_MSG_PERIOD 100 // in ms
-#define FSM_PERIOD 100 // in ms
-#define DEBOUNCE_PERIOD 2 // in units of FSM_PERIOD
-#define MOTOR_MSG_COUNTER_THRESHOLD (MOTOR_MSG_PERIOD)/(FSM_PERIOD)
+/**
+ * @def DEBOUNCE_PERIOD
+ * @brief Period of the debouncer in units of FSM_PERIOD
+ */
+#define DEBOUNCE_PERIOD 2
 
+/**
+ * Generate enum list for Gear_t
+*/
 #define FOREACH_Gear(GEAR) \
         GEAR(FORWARD_GEAR),   \
         GEAR(NEUTRAL_GEAR),  \
         GEAR(REVERSE_GEAR),   \
 
+/**
+ * @enum Gear_t
+ * @brief Enumerated list of gears
+*/
 typedef enum GEAR_ENUM {
     FOREACH_Gear(GENERATE_ENUM)
     NUM_GEARS,
 } Gear_t;
 
-// State Names
+/**
+ * @enum TritiumStateName_t
+ * @brief Enumerated list of states for the FSM
+*/
 typedef enum{
-    FORWARD_DRIVE,
-    NEUTRAL_DRIVE,
-    REVERSE_DRIVE,
-    RECORD_VELOCITY,
-    POWERED_CRUISE,
-    COASTING_CRUISE,
-    BRAKE_STATE,
-    ONEPEDAL,
-    ACCELERATE_CRUISE
+    /** Forward drive state */
+    FORWARD_DRIVE,   
+    /** Neutral drive state */
+    NEUTRAL_DRIVE,   
+    /** Reverse drive state */
+    REVERSE_DRIVE,   
+    /** Record velocity state (cruise) */
+    RECORD_VELOCITY, 
+    /** Powered cruise state (cruise) */
+    POWERED_CRUISE,  
+    /** Coasting cruise state (cruise) */
+    COASTING_CRUISE, 
+    /** Brake state */
+    BRAKE_STATE,     
+    /** One pedal state (regen) */
+    ONEPEDAL,        
+    /** Accelerate cruise state (cruise) */
+    ACCELERATE_CRUISE   
```

Comment:
Not sure if this type of commenting is neccessary just because it can feel a bit redundant but doesn't matter too much. Might be better to over document then under.

Reply:
Good stuff man! Keep it up!